src/gfx/.clang-format

```diff
@@ -0,0 +1,7 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
+TabWidth: 4
+AccessModifierOffset: -4
+UseTab: Always
+ColumnLimit: 160
+AlignEscapedNewlines: Left
```

Comment:
Why this sudden style change for this folder?

Reply:
Yeah let me fix that and make a PR into devel for that

Reply:
Resolved in #165